### PR TITLE
Create View for View Habit Event

### LINKED ISSUE #71: Create view for "View habit event"
[UI Design Mockup](https://github.com/CMPUT301F21T09/BudgetProjectName/wiki/User-Interface-Mockups#view-habit-event-details)

See #39

### PR DESCRIPTION
**Pull request by Docker-J**
## Description of Changes

Create View for View Habit Event

## Screenshots
Screenshot_1635462317

## Checklist

- [ ] Automated tests
- [x] Screenshots included (if necessary)
- [x] Code is well commented

Closes #71

### CODE REVIEW

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/ViewHabitEventActivity.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_view_habit_event);
+
+        // TODO: Get Targeted Habit Event
+
+        TextView habitTitle = findViewById(R.id.view_habit_event_habit_title);
+        TextView habitEventLocation = findViewById(R.id.view_habit_event_habit_event_location);
+        TextView habitEventDescription = findViewById(R.id.view_habit_event_habit_event_description);
+        TextView habitEventDate = findViewById(R.id.view_habit_event_habit_event_date);
+        // TODO: Set Proper Name of Habit, Location of Habit Event to the TextViews
```
[the remark this answers is not in the document]
Comment (j-vent) [reply src/app/src/main/java/com/cmput301f21t09/budgetprojectname/ViewHabitEventActivity.java:25]: Oh yeah you're right this view is for displaying information from the backend, my bad.

**Review by j-vent — DISMISSED** [review]
Looks good to me

**Review by cheetiantey — APPROVED** [review]
Everything looks great! I know that we've decided that we have to prioritize (functionality->tests->UI), but if you would like to add the back button, it's pretty easy. I've created a [gist](https://gist.github.com/cheetiantey/d1300cba6cc2f8c66154f919717e222c#file-java). You just have to include line 9 and lines 12-22 in your ViewHabitEventActivity.java file. :)

**Comment by Docker-J** [comment]
> Everything looks great! I know that we've decided that we have to prioritize (functionality->tests->UI), but if you would like to add the back button, it's pretty easy. I've created a [gist](https://gist.github.com/cheetiantey/d1300cba6cc2f8c66154f919717e222c#file-java). You just have to include line 9 and lines 12-22 in your ViewHabitEventActivity.java file. :)

As we need to use custom toolbar to set the title text in center, I'll just leave this as it is. Actually I am working on it

image

**Comment by cheetiantey** [comment]
> > Everything looks great! I know that we've decided that we have to prioritize (functionality->tests->UI), but if you would like to add the back button, it's pretty easy. I've created a [gist](https://gist.github.com/cheetiantey/d1300cba6cc2f8c66154f919717e222c#file-java). You just have to include line 9 and lines 12-22 in your ViewHabitEventActivity.java file. :)
> 
> As we need to use custom toolbar to set the title text in center, I'll just leave this as it is. Actually I am working on it
> 
> image

Sounds good!

**Review by MacMalainey — APPROVED** [review]

**Pull request merged**